cg/src/domains/subjects/subjects.service.js

```diff
@@ -103,7 +95,6 @@ class SubjectsService {
   async _createNewSubject(personalData, trx, subjectId) {
     const encryptionKey = generateClientKey();
     const encryptedPersonalData = encryptForStorage(JSON.stringify(personalData), encryptionKey);
-
```

Comment:
Not sure why you removed all the white-spacing. Just like with writing e-mails or documents, whitespace between paragraphs improves readability. I don't mind this now, but whitespaces don't cost us much, but where it makes sense, I'd like to keep them.
This isn't a necessary change though.

Reply:
Not all whites-pacing, just the ones between lines of the same blocks of code. There's spaces after db queries, functions, tests and other blocks of code